.github/workflows/codeql.yml

```
@@ -34,4 +37,4 @@ jobs:
         run: |
           make -j BOLOS_SDK=${{ matrix.sdk }}
       - name: Perform CodeQL Analysis
-        uses: github/codeql-action/analyze@v2
+        uses: github/codeql-action/analyze@v3
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)